cupy_setup_build.py

```diff
@@ -418,12 +418,21 @@ def _nvcc_gencode_options(cuda_version):
     if cuda_version >= 8000:
         arch_list += ['compute_60']
 
+    if cuda_version >= 9000:
+        arch_list = [('compute_35', 'sm_35'),
```

Comment:
Why do not contain `compute_30`?
How about `arch_list += ['compute_70']`?

Reply:
I will add `compute_30` to the list.
I suppose your question is "why are those like `('compute_70', 'sm_70')` added to the list instead of just adding `('compute_70')`?". This is because cubin is not generated at compile time if cubin version (sm_XX) is not specified with `code=` clause.

Reply:
In my understand, CUDA generates cubin code automatically. And, CUDA caches it. Is it correct?
Then, We can reduce compile time at installation time.

Reply:
When you launch a kernel and there is no cubin format of that kernel, CUDA runtime loads the PTX format of the kernel and JIT-compile it to cubin format. That generated cubin kernel will be cached thus JIT-compilation is one-time. However the cubin kernels generated by nvcc are basically better than that generated by JIT-compilation. Thus, we would recommend to build cubin format of kernels at installation time when possible. Please see http://docs.nvidia.com/cuda/volta-compatibility-guide/index.html#building-applications-with-volta-support.